lib/sandbox.js

```diff
@@ -1618,9 +1630,9 @@ function sandBox(script, name, verbose, debug, context) {
         getState:       function (id, callback) {
             if (typeof callback === 'function') {
                 if (id.indexOf('.') === -1) {
-                    adapter.getState(id, callback);
+                    adapter.getState(id, (err, state) => callback(err, context.convertBackStringifiedValues(id, state)));
```

Comment:
If this is going to be used more often, this callback wrapper should be refactored into a separate method.

Reply:
only those some places ... would leave for now

Reply:
PS: But yes I also thought that